README.md

```diff
@@ -53,10 +53,7 @@ Project Structure
 ```
 container-spi (contains no dependencies on a specific web container)
 container
-  catalina (base Tomcat/JBoss Web container implementation, based on Tomcat 5.5)
-  catalina-standalone (contains ModClusterListener, used for standalone Tomcat/JBoss Web installations)
-  jbossweb (JBoss Web container implementation, all versions)
-  tomcat6 (Tomcat 6.0 container implementation)
+  catalina (base for Tomcat container implementations, based on Tomcat 7)
   tomcat7 (Tomcat 7.0 container implementation)
```

Comment:
Yes, removed. Although I intended to rework this afterwards as part of MODCLUSTER-439.